docs/internals/contributing/writing-documentation.txt

```diff
@@ -351,15 +351,15 @@ Documenting new features
 Our policy for new features is:
 
     All documentation of new features should be written in a way that
-    clearly designates the features are only available in the Django
+    clearly designates the features that are only available in the Django
     development version. Assume documentation readers are using the latest
     release, not the development version.
 
 Our preferred way for marking new features is by prefacing the features'
 documentation with: "``.. versionadded:: X.Y``", followed by a mandatory
 blank line and an optional description (indented).
 
-General improvements, or other changes to the APIs that should be emphasized
+General improvements or other changes to the APIs that should be emphasized
```

Comment:
Can you explain this change? We're not necessarily glossing "general improvements," and if we were, there would need to be another comma on the other side.

Reply:
I don't like to use a comma with "or" very often. BTW I would revert it if you say!

Reply:
Ha, I read this proposed change backwards. I agree with your edit. I was confused and thought you were adding a comma. My mistake.